bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/JSONParser.java

```diff
@@ -21,7 +21,7 @@
 import org.ballerinalang.jvm.types.BArrayType;
 import org.ballerinalang.jvm.types.BTypes;
 import org.ballerinalang.jvm.types.TypeTags;
-import org.ballerinalang.jvm.util.exceptions.BallerinaException;
+import org.ballerinalang.jvm.util.exceptions.JBallerinaException;
```

Comment:
This is temporary change to overcome using fully qualified names specially in native functions where we have bvm and jvm target support code in same class. . ex. Clone.java